susemanager-utils/susemanager-sls/salt/bootstrap/remove_traditional_stack.sls

```diff
@@ -35,12 +43,15 @@ remove_traditional_stack:
 {%- elif grains['os_family'] == 'Debian' %}
       - apt-transport-spacewalk
 {%- endif %}
-      - osad
-      - osa-common
-      - spacewalksd
-      - rhncfg
-      - rhnlib
-      - rhnmd
```

Comment:
I guess this state now needs an `onlyif` clause here to prevent it from being applied on Servers or Proxies (or some other mechanism to achieve the same effect).

Reply:
same as above. These packages should go away also on Server and Proxy when they are salt managed

Reply:
This state has it